pages/index.js

```diff
@@ -22,35 +22,61 @@ export default function Home() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main>
-        <Headline>Tutorial Maker</Headline>
-        <HomeButtons>
-          {session ? (
-            <Button
-              onClick={() =>
-                signOut({
-                  callbackUrl: "/",
-                })
-              }
-            >
-              <TbLogout style={iconStyle} />
-              Sign out
-            </Button>
-          ) : (
-            <Button isPrimary onClick={() => signIn()}>
-              <TbLogin style={iconStyle} />
-              Sign in
-            </Button>
-          )}
+      <LogoContainer>
+        <h1>
+          <object
+            type="image/svg+xml"
+            data="/assets/hamster logo v2.svg"
+            aria-label="How to bathe a hamster – Your tutorial maker | logo"
+          >
+            How to bathe a hamster – Your tutorial maker | logo
+          </object>
+        </h1>
+      </LogoContainer>
 
-          <Link href="/tutorials" passHref>
-            <Button>
-              <TbBook2 style={iconStyle} />
-              Library
-            </Button>
-          </Link>
-        </HomeButtons>
-      </main>
+      <StyledSection>
+        <Subline style={{ color: "var(--darktext)" }}>
```

Comment:
You could use styled components to put your `Subline` variations into new styled components instead of reverting back to imperatively styling the component via the style prop.

Reply:
Thanks for you feedback, @Kheirah ! At first I thought that for only one changed prop it is not worth to make a separate component but you are absolutely right, for clarity it is much better. Besides, this way I can use it everywhere.